taxonomies/labels.txt

```diff
@@ -19142,6 +19147,14 @@ origins:en: en:france
 label_categories:en: en:Honey
 ingredients:en: en:honey
 
+#<en:supply chain labels
+#<en:certified labels
+description:en:True Source Honey is a voluntary system of traceability and authenticity testing for those participants who wish to demonstrate through an independent third party that their sourcing practices are in full compliance with U.S. and international trade laws.
+en:True Source Certified
```

Comment:
This line must be the first line, we can't have properties like description:en: above it.